site/src/components/message.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Message
+description: The message component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - message
+---
```

Comment:
Is a Figma file available for this component? Please link it here if it is (below is example of how to do that), if not - please ask the designer you're working with to create it. If there are any problems/questions on how to do that - reach out to me or Sanwal :)

```
figmaEmbed: {figma-file-url} allowfullscreen
```

Reply:
added

Reply:
updated